### PR TITLE
Mobile page design

### PR DESCRIPTION
**Pull request by aishatpopoola**
This PR does the following

- Creates mobile design for the home section
- Creates mobile design for the about section
- Creates mobile design for the skills section
- Creates mobile design for the qualification section
- Creates mobile design for the services section
- Creates mobile design for the portfolio section
- Creates mobile design for the contact-us section
- Add mobile design for the footer section

### CODE REVIEW

File: css/mobile_screen.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.container {
+  margin-left: var(--mb-1);
+  margin-right: var(--mb-1);
```
Comment (adejam) [inline css/mobile_screen.css]: Use shorthand if possible

File: css/reset.css [hunk 2]
```diff
@@ -1 +1,138 @@
...
+}
+.change-theme{
+  font-size: 1.25rem;
+  color: var(--title-color);
+  margin-right: var(--mb-1);
+  cursor: pointer;
+}
+.change-theme:hover{
+  color: var(--first-color);
+}
```
Comment (adejam) [inline css/reset.css]: This property is repeated in many places. We should make our code as [dry](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) as possible

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+// /*===== MENU SHOW =====*/
+// /* Validate if constant exists */
+
+// const navToggler = () => {
+//     navMenu.classList.add('show-menu')
+// }
+// if (navToggle) {
+//     navToggle.addEventListener('click', navToggler)
+// }
```
Comment (adejam) [inline js/main.js]: remove commented js file

**Review by adejam — APPROVED** [review]
# LGTM

- Lets make the requested changes and merge.

Goodluck on your next milestone

File: css/mobile_screen.css (hunk 1 above)
Comment (aishatpopoola) [reply css/mobile_screen.css]: Thank you, i have made the changes

File: js/main.js (hunk 3 above)
Comment (aishatpopoola) [reply js/main.js]: ok, thanks

File: css/reset.css (hunk 2 above)
Comment (aishatpopoola) [reply css/reset.css]: ok I will be more careful, thanks

**Comment by aishatpopoola** [comment]
Thanks for the review, i have made changes were required.

**Pull request merged**